components/browser/errorpages/src/main/res/values/strings.xml

```diff
@@ -32,6 +33,7 @@
 
 
     <string name="error_malformedURI_title">The address isn’t valid</string>
```

Comment:
I should likely file a separate issue, but I notice that the formatting of these names is inconsistent. a few examples from this file:

- `error_connectionfailure_title`
- `error_malformedURI_title`
- `error_redirectLoop_title`
- `error_unsupportedprotocol_title`
- `error_sslhandshake_title`
- `errorpage_title`
- `error_generic_title`
- `error_security_bad_cert_title`
- `error_connection_failure_title`
- `error_safe_browsing_malware_uri_title`

Reply:
I'd like to fix this now, but it seems like there's a name collision if I remove the camel casing and for `error_connectionfailure_title` and it's not yet clear to me why this string resource is needed.

Reply:
In order to keep these strings in and fix this issue, I'm just going to append `_alternative` to the end of the duplicates that aren't being used by the `ErrorType` enum, and we can still have these translated for now.